packages/shared/src/components/modals/FeedFiltersModal/FeedFiltersIntroModal.tsx

```diff
@@ -30,16 +37,58 @@ type FeedFiltersIntroModalProps = FeedFiltersProps &
   ModalFooterProps &
   ModalProps;
 
-const footerClass = {
-  introTest1: 'justify-center',
-  introTest2: 'justify-between',
-};
+const getAnalyticsEvent = (
+  eventName: string,
+  copy: string,
+  type: string,
+  targetType: string,
+): Partial<AnalyticsEvent> => ({
+  event_name: eventName,
+  target_type: targetType,
+  target_id: type,
+  feed_item_title: copy,
+});
 
 const IntroModalFooter = ({
   feedFilterOnboardingModalType,
   onRequestClose,
   onOpenFeedFilterModal,
 }: ModalFooterProps) => {
+  const { trackEvent } = useContext(AnalyticsContext);
+
+  useEffect(() => {
+    trackEvent(
+      getAnalyticsEvent(
+        'impression',
+        'skip button',
```

Comment:
I think the right value here is `open feed filter button` by looking a the other parameters. Can you confirm?
I also see that in other places (outside the scope of this change), we used the value `my feed button` for indicating a click on the CTA that opens the feed filters. Can you please confirm as well and if so please change wherever necessary